Lexdao-Constitutionv02.md

```diff
@@ -0,0 +1,160 @@
+
+**Section 3.6 - Voting.** Unless otherwise required or designated in writing, each member in good standing shall have one and only one vote on each matter submitted to a vote of members. Voting by manual proxy shall not be permitted, programmatic proxy voting may be used with the consent of the DAO unless otherwise required by law or by this agreement, issues shall be decided by the Designated Smart Contract.  Records of membership shall be associated with an Account Address and be made accessible via Designated Smart Contract on the Designated Blockchain Network.
+
+ # Article IV - Decentralized Governance
+
+**Section 4.1 - Powers and duties.** Except as to matters reserved to members by law or by this constitution, all LexDAO powers shall be exercised by or under the authority of LDC members or such agents or designees approved by LDC Members through Designated Smart Contract voting.
+
+# Article V - Governance Shares and Capital Shares
+
+**Section 5.1 - Issuance and terms.** It is noted that different Designated Smart Contracts have different manners of managing governance shares (“LDC”) versus contributed capital and efforts resulting in retains to LexDAO (“Loot Shares”).  Currently, LDC members have no rights of redemption of any working capital, or token assets, held within LexDAO.  In the future, to evidence capital or contributions of efforts provided by members, the Co-op may issue “Loot Shares”. Loot Shares may be issued only to persons eligible for and admitted to membership in LexDAO. Loot Shares shall be considered issued upon their issue and they need not be evidenced by certificates.  Loot Shares shall be entitled to no dividend or other monetary return on contributed capital and may not be used to secure loans or other obligations of members. Loot Shares shall be subject to assessment for the reasonable capital needs of LexDAO, as determined by mutual agreement.  Loot Shares (if issued) should be considered to be similar to “retains” and are envisioned as a manner of tracking contributions made to the nonprofit.  It is not anticipated that LexDAO will ever turn a “profit” as excess capital will be paid out via charitable donations, grants, or other forms of support meant to carry out the Mission Statement.
+
+**Section 5.2 - Redemption.** Upon request following termination of LexDAO membership or via a future Smart Contract Mechanism to remove capital via a burn of Loot Shares, the Loot Shares shall be redeemed via a mechanism to be determined by LDC members.  A “Rage Quit” is a request to burn both LDC Membership token and Loot Shares. A “Loot Quit”, is simply to burn and withdraw Loot Shares as return of retains.  Loot Shares (if ever created) shall be redeemable at the approved Designated Smart Contract less a reasonable processing fee (if any).
+
+# Article VI - Guild Kick/Member Removal and Exit
```

Comment:
Reference to previous comment on "member disputes" this might be better served within the Membership section or directly after.